documentation/mDNS_Support.md

```diff
@@ -20,17 +20,18 @@ Our sample file `physical-web-url.service` looks like this:
 <!DOCTYPE service-group SYSTEM "avahi-service.dtd">
 
 <service-group>
-  <name replace-wildcards="yes">http://www.mycompany.com/xyz.html</name>
+  <name replace-wildcards="yes">Physical Web on %h</name>
   <service>
-    <host-name>www.mycompany.com</host-name>
-    <type>_http._tcp</type>
+    <type>_urlbookmark._tcp</type>
```

Comment:
The URL was split into the field 'type' (http vs https), host-name (www.mycompany.com), port (80) and path (/xyz.html), which result in http://www.mycompany.com:80/xyz.html
The name was just to help people choose a unique name and could be anything else.